src/plugins/vis_type_drilldown/public/drilldown_options.tsx

```diff
@@ -12,23 +12,72 @@ import {
   EuiFlexItem,
   EuiFieldText,
   EuiAccordion,
+  EuiSuperSelect,
+  EuiText,
 } from '@elastic/eui';
-import { FormattedMessage } from '@osd/i18n/react';
 
 import { VisOptionsProps } from 'src/plugins/vis_default_editor/public';
-import { DrilldownVisParams } from './types';
+import { useOpenSearchDashboards } from 'src/plugins/opensearch_dashboards_react/public';
```

Comment:
Looking at most other imports, I think this needs to be relative:

```suggestion
import { useOpenSearchDashboards } from '../../opensearch_dashboards_react/public';
```